### PR TITLE
Gv distinguishing particles in brightfield tutorial

### PR DESCRIPTION
**Pull request by giovannivolpe**

### CODE REVIEW

File: tutorials/distinguishing_particles_in_brightfield_tutorial.ipynb [hunk 1]
```diff
@@ -56,15 +56,15 @@
    "source": [
     "## 2. Define the particles\n",
     "\n",
-    "For this example, we consider circular disk particles of two different sizes. The particles are instances of the class `Sphere`, which takes the following inputs:\n",
+    "We consider circular disk particles of two different sizes. The particles are instances of the class `Sphere`, which takes the following inputs:\n",
```
Comment (BenjaminMidtvedt) [inline tutorials/distinguishing_particles_in_brightfield_tutorial.ipynb]:
```suggestion
    "We consider spherical particles of two different sizes. The particles are instances of the class `Sphere`, which takes the following inputs:\n",
```

File: tutorials/distinguishing_particles_in_brightfield_tutorial.ipynb [hunk 2]
```diff
@@ -75,21 +75,21 @@
...
     "\n",
     "large_particle = Sphere(\n",
     "    position=lambda: np.random.rand(2) * 256,\n",
-    "    z=lambda:np.random.rand() * 20 - 10,\n",
-    "    position_unit=\"pixel\", # Defaults to meter\n",
-    "    radius=lambda: np.random.rand() * 0.1e-6 + 0.4e-6,\n",
+    "    z=lambda: -10 + np.random.rand() * 20,\n",
+    "    position_unit=\"pixel\",\n",
+    "    radius=lambda: 400e-9 + np.random.rand() * 100e-9,\n",
     "    refractive_index=lambda: 0.12 + (0.1j + np.random.rand() * 0.05j),\n",
```
Comment (BenjaminMidtvedt) [inline tutorials/distinguishing_particles_in_brightfield_tutorial.ipynb]:
```suggestion
    "    refractive_index=lambda: 1.45 + (0.1j + np.random.rand() * 0.05j),\n",
```

File: tutorials/distinguishing_particles_in_brightfield_tutorial.ipynb [hunk 3]
```diff
@@ -75,21 +75,21 @@
    "source": [
     "small_particle = Sphere(\n",
     "    position=lambda: np.random.rand(2) * 256,\n",
-    "    z=lambda:np.random.rand() * 20 - 10,\n",
-    "    position_unit=\"pixel\", # Defaults to meter\n",
-    "    radius=lambda: np.random.rand() * 0.05e-6 + 0.2e-6,\n",
+    "    z=lambda: - 10 + np.random.rand() * 20,\n",
+    "    position_unit=\"pixel\",\n",
+    "    radius=lambda: 200e-9 + np.random.rand() * 50e-9,\n",
     "    refractive_index=lambda: 0.12 + (0.1j + np.random.rand() * 0.05j),\n",
```
Comment (BenjaminMidtvedt) [inline tutorials/distinguishing_particles_in_brightfield_tutorial.ipynb]:
```suggestion
    "    refractive_index=lambda: 0.45 + (0.1j + np.random.rand() * 0.05j),\n",
```

**Comment by giovannivolpe** [comment]
I've now finished the changes to the tutorial. It should be ready now.

Please, add the comments about how to set the refractive index of the medium in section 2 of this tutorial.

**Comment by BenjaminMidtvedt** [comment]
@giovannivolpe Wouldn't it be more natural in section 3, where we're setting the optics?

**Comment by giovannivolpe** [comment]
@BenjaminMidtvedt Ok to do it in section 3.

**Comment by BenjaminMidtvedt** [comment]
@giovannivolpe Please see this commit

**Pull request merged**